docs/translations/ro/scripting/callbacks/OnActorStreamIn.md

```diff
@@ -0,0 +1,45 @@
+---
+id: OnActorStreamIn
+titlu: OnActorStreamIn
+descriere: Acest apel invers este apelat atunci când un actor este transmis în flux de către clientul unui jucător.
+tags: []
+---
```

Comment:
I _think_ these property names (title, description) need to be in english, @Southclaws?